src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbTestRunner.java

```diff
@@ -485,93 +485,93 @@ void run() {
         log.info("END test " + testDescription);
     }
 
-    private List<MTRecord> getExpectedMTRecords() {
+    private List<MtAmazonDynamoDb.MtRecord> getExpectedMtRecords() {
         return ImmutableList.of(
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
                 .withContext("ctx1")
-                .withTableName(getPrefix() + "MTAmazonDynamoDBTestRunner1")
+                .withTableName(getPrefix() + "MtAmazonDynamoDbTestRunner1")
                 .withEventName(INSERT.name())
                 .withDynamodb(new StreamRecord()
                     .withKeys(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue")))
                     .withNewImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue1")))),
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
                 .withContext("ctx2")
-                .withTableName(getPrefix() + "MTAmazonDynamoDBTestRunner1")
+                .withTableName(getPrefix() + "MtAmazonDynamoDbTestRunner1")
                 .withEventName(INSERT.name())
                 .withDynamodb(new StreamRecord()
                     .withKeys(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue")))
                     .withNewImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue2")))),
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
```

Comment:
why is this expanded to explicitly denote the outer class (i.e., "MtAmazonDynamoDb.MtRecord" instead of just "MtRecord")?